mockingcase.test.js

```diff
@@ -21,4 +21,23 @@ describe('mockingcase', () => {
     expect(mOcKiNgCaSe(input)).toHaveLength(input.length);
     expect(mOcKiNgCaSe(input, options)).toHaveLength(input.length);
   });
+
+  test('Should convert even index letters to uppercase and odd index letters to lowercase', () => {
+    const input = 'hello world';
+    const options = {
+      firstUpper: true
+    };
+    const expectedOutput = 'HeLlO WoRlD';
+    expect(mOcKiNgCaSe(input, options)).toEqual(expectedOutput);
+  });
+
+  test('Should only modify letters', () => {
```

Comment:
Can you change the name of the test because of this name already exists and can be confusing?!